src/controller/RacingGameController.js

```diff
@@ -0,0 +1,72 @@
+const RacingGame = require('../model/RacingGame');
+const inputView = require('../view/inputView');
+const outputView = require('../view/outputView');
+const validate = require('../utils/validation');
+const { SIGN, ERROR } = require('../utils/constants');
+
+class RacingGameController {
+  #inputView;
+  #outputView;
+  constructor() {
+    this.#inputView = inputView;
+    this.#outputView = outputView;
+    this.initiate();
+  }
+
+  initiate() {
+    this.#readCarNames();
+  }
+
+  #readCarNames() {
+    this.#inputView.readUserInputNames((input) => {
+      const names = input.split(SIGN.comma);
+      this.#handleCarNamesException(names);
+    });
+  }
+
+  #readTrialCounts(game) {
+    this.#inputView.readUserInputTrialCounts((input) => {
+      this.#handleTrialCountsException(input, game);
+    });
+  }
+
+  #handleCarNamesException(names) {
+    try {
+      this.#validateCarNames(names);
+      const racingGame = new RacingGame(names);
+      this.#readTrialCounts(racingGame);
+    } catch (error) {
+      this.#outputView.printError(ERROR.userInputNameInvalid);
+      this.#readCarNames();
+    }
+  }
```

Comment:
```suggestion
  #handleCarNamesInput(input) {
    try {
      const names = this.#extractCarNamesByInput(input); // 이름은 생각이 잘 안나네...
      this.#validateCarNames(names);
      // 1
      this.#readTrialCounts(new RacingGame(names));  // 2
    } catch (error) {
      this.#outputView.printError(ERROR.userInputNameInvalid);
      this.#readCarNames();
    }
  }
```
1. 개행으로 가독성을 높일 수도 있어. input을 validate하는 로직과 valid한 input을 사용하는 로직은 분리된 과정이라고 할 수 있으니까 개행으로 나눠주는게 좋아. 그러면 읽는 입장에서 편안하지 한 뭉탱이당 하나의 역할이라고 생각하면 되니까.

2. 변수가 늘어난다는 건 관리해야할 포인트가 늘어난다는 거야. 따라서 가독성 상 문제가 있는게 아니라면 지역 변수는 없는게 좋아. 
`const racingGame = new RacingGame(names);` 에서 사실 상 racingGame = RacingGame으로 똑같은 정보가 반복되잖아. 지역변수로 만들 필요가 없는 거나 마찬가지지